server/services/visitService.ts

```diff
@@ -59,6 +59,29 @@ export default class VisitService {
     return visit
   }
 
+  async updateVisit({
+    username,
+    applicationReference,
+    applicationMethod,
+    allowOverBooking,
+  }: {
+    username: string
+    applicationReference: string
+    applicationMethod: ApplicationMethodType
+    allowOverBooking: boolean
+  }): Promise<Visit> {
+    const token = await this.hmppsAuthClient.getSystemClientToken(username)
+    const orchestrationApiClient = this.orchestrationApiClientFactory(token)
+
+    const visit = await orchestrationApiClient.updateVisit(
+      applicationReference,
+      applicationMethod,
```

Comment:
Re my earlier comment, I see why it has a different name now.

Reply:
Also, the call to this service passes an object but when you call the client you're passing individual arguments? For consistency, I'd pass an object here but not sure how the rest of the code looks.

Reply:
This is essentially a duplicate of the book visit endpoint, I think keeping them structured and variables named the same is beneficial in this case as it's very clear when looking at them side by side that they have same input / output